lib/ansible/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -44,6 +44,15 @@
     default: false
     type: bool
 
+  boot_type:
+    description:
+      - Boot method
+    default: bootscript
```

Comment:
I think the default should be local @CedricCabessa Could you confirm?

Reply:
Actually the default depend on the offer, for exemple C1 and C2 (baremetal) use bootscript by default but DEV (virtualized) use local
It is better to not specifiy this parameter and let the API choose the right one